doc/source/conf.py

```diff
@@ -111,9 +111,11 @@ def setup(app):
 # ones.
 extensions = [
     'sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.napoleon', 'sphinx.ext.intersphinx', 'sphinx.ext.todo',
-    'sphinx.ext.coverage', 'sphinx.ext.viewcode'
+    'sphinx.ext.coverage', 'sphinx.ext.viewcode', 'sphinx.ext.autosectionlabel'
```

Comment:
How does this affect the rest of the documentation?  A little description of why this is here (just in the PR comments) would be useful.

Reply:
That helped me to tag titles from other pages, if I remember correctly. It was long back I added that. But am sure it was for cross referencing.